config/development.json

```
@@ -175,6 +175,7 @@
 		"server-side-rendering": true,
 		"settings/newsletter-settings-page": true,
 		"settings/security/monitor": true,
+		"settings/start-over/site-reset": true,
```

Can we use something more descriptive instead?

```suggestion
		"settings/self-serve-site-reset": true,
```